css/style.css

```diff
@@ -0,0 +1,643 @@
+  margin: 2.5rem 0;
+  box-sizing: border-box;
+  width: 100%;
+  border: 1px solid #d3d3d3;
+}
+
+.p-64 {
+  padding-top: 4rem;
+}
+
+@media screen and (min-width: 768px) {
+  .headline {
+    padding: 0% 10%;
+
+    /* height: 100vh; */
+    background-image: url(../images/main_big.png);
+  }
+
+  .about {
+    padding: 0 10%;
+    background-image: url(../images/main-bg.png);
+    background-repeat: no-repeat;
+  }
+
+  .headline .headline-content,
+  .about-contents,
+  .about .supportive-contents {
+    width: 80%;
+  }
+
+  .about .about-contents,
+  .feature .logo-info {
+    margin: 0 auto;
+  }
+
+  .headline .headline-content .desc-box {
+    width: 80%;
+  }
+
+  .headline .headline-content h3 {
+    font-size: 1.5rem;
+  }
+
+  .headline .headline-content h1 {
+    max-width: 94%;
+    text-align: left;
+    line-height: 3.5rem;
+  }
+
+  @media screen and (min-width: 992px) {
+    .headline .headline-content h1 {
+      /* width: 65%; */
+      text-align: left;
+      font-size: 3.25rem;
+      line-height: 3.75rem;
+    }
+  }
+
+  .about .about-contents .supportive-contents .about-desc-box {
+    padding: 2rem;
+  }
+}
+
+.feature .logo-info,
+.feature .feature-logo,
+.feature .logo-design,
+.feature .namm-logo,
+.feature .pst-occurence,
+.feature .pst-occurence .pst-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.feature.bg-white,
+.feature .pst-occurence {
+  border-top: 1px solid #d3d3d3;
+}
+
+.feature .pst-occurence {
+  padding: 1.5rem;
+}
+
+.feature .logo-info {
+  padding: 2.5rem 1.5rem;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.text-justify {
+  text-align: justify;
+}
+
+.sm-fnt {
+  font-size: 0.83333125rem;
+}
+
+.feature .logo-info .namm-logo {
+  margin: 2.5rem 0 0 0;
+  border: 1px solid #d3d3d3;
+  width: 100%;
+  padding: 2.5rem 0;
+  box-sizing: border-box;
+}
+
+.feature .pst-occurence .card-container .card {
+  position: relative;
+  text-align: center;
+}
+
+.feature .pst-occurence .card-container .card .card-body-block {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.feature .pst-occurence .pst-container .card-container .card .card-images {
+  opacity: 0.5;
+  height: 161.27px;
+  width: 332.5px;
+}
+
+.fnt-22 {
+  font-size: 1.25rem;
+}
+
+.feature .pst-occurence .card-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1.5rem;
+  gap: 1.25rem;
+}
+
+@media screen and (min-width: 768px) {
+  .feature .logo-info .namm-logo {
+    width: 30%;
+    padding: 1rem 0;
+  }
+
+  .feature .pst-occurence .card-container {
+    flex-direction: row;
+    width: 80%;
+  }
+
+  .feature .logo-info,
+  .feature .pst-occurence {
+    padding-right: 4.9rem;
+    padding-left: 4.9rem;
+  }
+}
+
+.program,
+.featured-speakers {
+  padding: 1.5rem 0.75rem;
+  display: flex;
+  flex-direction: column;
+  background-image: url(../images/fill-bg.png);
+  align-items: center;
+}
```

Comment:
- [ ] For the main program section in the desktop view, please increase the value of the font-size of the `h6` header and make the icons larger and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/68796232/170270964-323d6df4-d959-4938-9f08-00058d180245.png) | ![image](https://user-images.githubusercontent.com/68796232/170270885-e1855929-3007-4135-a805-d27f17b86daa.png)  |

Reply:
- [x] this issue is well resolved :heavy_check_mark: